### DISCUSSION TITLE
Video DownloadHelper names videos downloaded with numbers instead of their real names.

### DISCUSSION THREAD

**Discussion by Moizevitch** [aclap-dev/video-downloadhelper#722] [Help and Q&A]
### Your question

Video DownloadHelper names videos downloaded with numbers instead of their real names. Is there a way to solve this, please? Thanks!

### Addon & CoApp version and details

Video DownloadHelper     Version 9.0.1.22

**Comment by GrampaWildWilly** [comment]
Let's have the URL of a specific web page on which this happens.  Not the name of a web site, a specific URL of a web page where a video is hosted.  If you are seeing this "everywhere," give at least one example of such a web page.  It might also help if you show a couple of screenshots showing the way the file name changes from one value to another.
